src/useUser.js

```diff
@@ -0,0 +1,28 @@
+import { useReducer } from 'react';
+
+// ENUM !!!!
```

Comment:
Probably shouldn't keep comments like this :joy:

Reply:
Oopsie. Forgot about that 🤣